api/src/Authentication/Type/OIDC.php

```diff
@@ -18,14 +18,20 @@ function __construct() {
         curl_setopt($ch, CURLOPT_HEADER, 0);
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
         $response = curl_exec($ch);
+
+        if (curl_errno($ch) || curl_getinfo($ch, CURLINFO_HTTP_CODE != 200)) {
+            error_log("Failed to connect to OIDC discovery endpoint");
```

Comment:
would it be worth logging the errno or http code at this point too?

Reply:
Since this is an open, relatively simple endpoint, I believe errors will amount to either 404s, 500s or timeouts (the latter being the most likely), so I think it's not as pivotal in this instance. However, since it's easy to add and there are no drawbacks, I'll add it